common/changes/@rushstack/eslint-config/ts-4.8_2022-09-26-07-14.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/eslint-config",
+      "comment": "Add support for TypeScript 4.8.",
```

```suggestion
      "comment": "Upgraded @typescript-eslint dependencies to 5.30.x to enable support for TypeScript 4.8"
```

Upgrading these dependencies is significant because often such "minor" upgrades can bring changes to behavior of ESLint rules, removal of formerly "deprecated" ESLint rules, or breaking changes to config schemas.  We really should review the `@typescript-eslint` changelogs to understand what was changed, or at least trying building a large monorepo with this upgraded ruleset.